pipelines/genomics/setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# [wf] execute setup stage
+
+# [wf] dc_worshop folder
+mkdir dc_workshop
+
+# [wf] Moving into the new folder
+cd dc_workshop
+
+# [wf] Make directory structure
+mkdir data
+mkdir data/ref_genome
+mkdir data/untrimmed_fastq
+mkdir data/trimmed_fastq
+mkdir results
+mkdir results/sai
+mkdir results/sam
+mkdir results/bam
+mkdir results/bcf
+mkdir results/vcf
+mkdir docs
+
+# Gathering all required files.
+curl -o data/ref_genome/ecoli_rel606.fasta -0L https://osf.io/vua9t/download
+curl -o data/trimmed_fastq/SRR097977.fastq_trim.fastq -0L https://osf.io/w7gp2/download
+
+# Gathering all the required tools.
+curl -o tbwa.tar.gz2 -0L https://sourceforge.net/projects/bio-bwa/files/latest/download
+tar -xvjf tbwa.tar.gz2
+cd bwa*
+./configure
+make
+make install
```

Comment:
we might need to change this so that this is not installing the tools on the system where the pipeline is running. An alternative is to do `./configure --prefix=` for all of the tools in this pipeline, and then modify the `PATH` variable on the `run.sh` command so that they point to the right `bin/` folder of each tool. Another one is to install inside a docker container.

Reply:
The docker option sounds fine. I'll put it in a docker container.

Reply:
And sure, I'll add validations. I can check whether or not the output files were correctly created and compare them with an example output from an execution on a Mac.

Reply:
cool! For the validation, this might help https://www.biostars.org/p/100901, looks like they explain a way of comparing two `vcf` files